sdks/java/maven-archetypes/examples/src/test/resources/projects/basic/archetype.properties

```diff
@@ -15,7 +15,7 @@
 #    limitations under the License.
 #
 package=it.pkg
-version=0.1-SNAPSHOT
+version=0.1
```

Comment:
This is the template pom.xml that will become part of the user's project, and thus should be separate from the versioning of Beam. The versioning for the user project is somewhat arbitrary, but it should represent "the first version for a new project", so 0.1 makes sense to me.

At any rate, we don't need this change coupled to this bug fix. I'll remove it here and file a separate bug.